specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -1,11 +1,11 @@
 == [[a235]]Security
 
 This chapter describes the security
-requirements for the Java™ Platform, Enterprise Edition (Java EE) that
-must be satisfied by Java EE products.
+requirements for the Jakarta™ Platform, Enterprise Edition (Jakarta EE) that
```

Comment:
Should we get rid of the TM as well?

Reply:
Jakarta is a trademark of the Eclipse Foundation see https://www.eclipse.org/legal/logo_guidelines.php

I would follow the rule in the above document `The appropriate trademark symbol (i.e. ™ or ®) must appear at both the first and most prominent use of Eclipse trademarks`